pkg/operation/etcdencryption/encryptionconfiguration_test.go

```diff
@@ -0,0 +1,311 @@
+package encryptionconfiguration
+
+import (
+	"testing"
+
+	apiserverconfigv1 "k8s.io/apiserver/pkg/apis/config/v1"
+)
+
+// kind: EncryptionConfiguration
+// apiVersion: apiserver.config.k8s.io/v1
+// resources:
+//   - resources:
+//     - secrets
+//     providers:
+//     - aescbc:
+//         keys:
+//         - name: key1553679720
+//           secret: t44dGAwGt73RMOSNwp4Z9QXadtnLvC4fZWgzS8Tjz+c=
+//     - identity: {}
+func TestCreateNewPassiveConfiguration(t *testing.T) {
```

Comment:
We use ginkgo/gomega BDD style for tests - can you please adapt the tests to this style?

Reply:
adapted tests